components/prism-ichigojam.js

```diff
@@ -0,0 +1,11 @@
+// according to the offical reference (EN)
+// https://ichigojam.net/IchigoJam-en.html
+Prism.languages.ichigojam = {
+	'string': /"(?:""|[!#$%&'()*,\/:;<=>?^_ +\-.A-Z\d])*"/i,
+	'comment': /\b(?:'|REM)(?:.*)?/i,
+	'number': /(?:(?:\b|\B[.-])(?:\d+\.?\d*)(?:E[+-]?\d+)?|(?:(?:\B`)(?:0|1)+))/i,
+	'keyword': /\b(?:BEEP|BPS|CASE|CLEAR|CLK|CLO|CLP|CLS|CLT|CLV|CONT|COPY|ELSE|END|FILE|FILES|FOR|GOSUB|GSB|GOTO|IF|INPUT|KBD|LED|LET|LIST|LOAD|LOCATE|LRUN|NEW|NEXT|OUT|RIGHT|PLAY|POKE|PRINT|PWM|REM|RENUM|RESET|RETURN|RTN|RUN|SAVE|SCROLL|SLEEP|SRND|STEP|STOP|SUB|TEMPO|THEN|TO|UART|VIDEO|WAIT)(?:\$|\b)/i,
+	'function': /\b(?:ABS|ANA|ASC|BIN|BTN|DEC|END|FREE|HELP|HEX|I2CR|I2CW|IN|INKEY|LEN|LINE|PEEK|RND|SCR|SOUND|STR|TICK|USR|VER|VPEEK|ZER)(?:\$|\b)/i,
+	'operator': /<[=>]?|>=?|[+\-*\/^=&]|\b(?:AND|EQV|IMP|NOT|OR|XOR)\b/i,
```

Comment:
The reference you linked seems to enumerate existing operators. Since you make a full component for IchigoJam, you might want to make a specific list and not just copy-paste the one from `basic`.
For example, the reference mentions `~` but does not seem to have `EQV` or `IMP`. Please double check this.

Reply:
Okay, I'm going to do a double check for all the parts before the next commit.